domain/src/main/java/com/teamagam/gimelgimel/domain/messages/poller/PolledMessagesProcessor.java

```diff
@@ -148,6 +152,7 @@ private void mapEntityToMessage(BaseMessageGeo message) {
         }
 
         private void displayGeoEntity(GeoEntity geoEntity) {
+            mGeoEntitiesRepository.add(geoEntity);
```

Comment:
Kinda duplicate don't u think? I mean, now every time I want to display an entity I must remember to first add it to the entities repo and then to show it using the displayed entities repo.
We have the same issue with messages I think...